bricksrc/quantities.py

```diff
@@ -1,109 +1,565 @@
+            },
+        },
     },
-    "Grains": {},
     "Power": {
-        "subclasses": {
+        OWL.sameAs: QUDTQK["Power"],
+        "subconcepts": {
             "Electric_Power": {
-                "subclasses": {
-                    "Apparent_Power": {},
-                    "Active_Power": {OWL.equivalentClass: BRICK["Real_Power"]},
-                    "Real_Power": {},
-                    "Reactive_Power": {},
-                    "Complex_Power": {},
+                OWL.sameAs: QUDTQK["ElectricPower"],
+                "subconcepts": {
+                    "Apparent_Power": {OWL.sameAs: QUDTQK["ApparentPower"]},
+                    "Active_Power": {
+                        OWL.sameAs: [QUDTQK["ActivePower"], BRICK["Real_Power"]],
+                    },
+                    "Real_Power": {
+                        OWL.sameAs: [QUDTQK["ActivePower"], BRICK["Active_Power"]],
+                    },
+                    "Reactive_Power": {OWL.sameAs: QUDTQK["ReactivePower"]},
+                    "Complex_Power": {OWL.sameAs: QUDTQK["ComplexPower"]},
                 },
             },
             "Peak_Power": {
+                SKOS.broader: QUDTQK.Power,
+                QUDT.applicableUnit: [UNIT.KiloW, UNIT.MegaW, UNIT.MilliW, UNIT.W],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L2I0M1H0T-3D0"],
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("PeakPower"),
                 SKOS.definition: Literal(
                     "Tracks the highest (peak) observed power in some interval"
                 ),
             },
-            "Thermal_Power": {},
+            "Thermal_Power": {
+                QUDT.applicableUnit: [UNIT.MilliW, UNIT.W, UNIT.KiloW, UNIT.MegaW],
+                QUDT.hasDimensionVector: QUDTDV["A0E0L2I0M1H0T-3D0"],
+                RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+                RDFS.label: Literal("ThermalPower"),
+                SKOS.broader: QUDTQK.Power,
+            },
         },
     },
-    "Cloudage": {},
+    "Cloudage": {
+        # TODO: define Okta?
+        QUDT.applicableUnit: [UNIT.Okta],
+        SKOS.definition: Literal(
+            "The fraction of the sky obscured by clouds when observed from a particular location"
+        ),
+        RDFS.isDefinedBy: URIRef(str(BRICK).strip("#")),
+        RDFS.label: Literal("Cloudage"),
+        SKOS.broader: QUDTQK.Dimensionless,
+    },
     "Current": {
-        "subclasses": {
+        "subconcepts": {
             "Electric_Current": {
-                "subclasses": {
-                    "Current_Angle": {},
-                    "Current_Magnitude": {},
-                    "Current_Imbalance": {},
-                    "Current_Total_Harmonic_Distortion": {},
-                    "Alternating_Current_Frequency": {},
+                OWL.sameAs: QUDTQK["ElectricCurrent"],
+                "subconcepts": {
+                    "Current_Angle": {
```

Comment:
Current_Angle is a subcomponent of Eletric_Current, but it is not a subconcept of Current.